Jetcaster/tv/src/main/java/com/example/jetcaster/tv/ui/player/PlayerScreen.kt

```diff
@@ -347,6 +348,10 @@ private fun PlayerControl(
                 modifier = Modifier
                     .size(JetcasterAppDefaults.iconButtonSize.large.intoDpSize())
                     .focusRequester(playPauseButton)
+                    .onGloballyPositioned {
```

Comment:
I believe the `onGloballyPositioned` modifier as `PlayPauseButton` is always in the viewport.